base/uk.ac.stfc.isis.ibex.ui.journalviewer.tests/src/uk/ac/stfc/isis/ibex/ui/journalviewer/tests/JournalViewModelTest.java

```diff
@@ -128,5 +129,55 @@ public void GIVEN_model_says_title_is_not_selected_THEN_viewmodel_says_title_is_
     	
     	assertFalse(viewModel.getFieldSelected(JournalField.TITLE));
     }
-
+    
+    @Test
```

Comment:
The aim of these tests is to test that we get the correct error message based on the dates and times set into the view model. I feel like it misses the aim as we are setting the error message and then testing we get the same thing back.

We should not do the line viewModel.setErrorMessage(actual)

Reply:
Fixed.